public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -105,206 +110,214 @@ code-example(language="bash").
   +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'class-selected-1', 'app.component.ts (setting the CSS class)')(format=".")
   :marked
-    Notice in the template that the `class.selected` is surrounded in square brackets (`[]`).
-    This is the syntax for a **property binding**, a binding in which data flows one way
+    Notice that `class.selected` is surrounded by square brackets (`[]`).
+    This is the syntax for a *property binding*, in which data flows one way
     from the data source (the expression `hero === selectedHero`) to a property of `class`.
   +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'class-selected-2', 'app.component.ts (styling each hero)')(format=".")
 
   .l-sub-section
     :marked
       Learn more about [property bindings](../guide/template-syntax.html#property-binding)
-      in the Template Syntax chapter.
+      in the Template Syntax page.
 
   :marked
-    The browser reloads our app.
-    We select the hero Magneta and the selection is clearly identified by the background color.
+    The browser reloads your app.
+    Click the hero Magneta. Your selection is clearly identified by the background color.
 
   figure.image-display
     img(src='/resources/images/devguide/toh/heroes-list-1.png' alt="Output of heroes list app")
 
   :marked
-    We select a different hero and the tell-tale color switches to that hero.
+    Click a different hero name and the selection color switches to that hero.
 
     Here's the complete `app.component.ts` as it stands now:
 
   +makeExample('toh-2/ts/app/app.component.ts', '', 'app.component.ts')
 
 .l-main-section
 :marked
-  ## The Road We’ve Travelled
-  Here’s what we achieved in this chapter:
+  ## The road you've travelled
+  Here's what you achieved in this page:
 
-  * Our Tour of Heroes now displays a list of selectable heroes
-  * We added the ability to select a hero and show the hero’s details
-  * We learned how to use the built-in directives `ngIf` and `ngFor` in a component’s template
+  * Your Tour of Heroes app now displays a list of selectable heroes.
+  * You added the ability to select a hero and show the hero's details.
+  * You learned how to use the built-in directives `ngIf` and `ngFor` in a component's template.
 
-  Run the <live-example></live-example> for this part.
+  **Try it out**. Here's a link to a <live-example></live-example>.
 
-  ### The Road Ahead
-  Our Tour of Heroes has grown, but it’s far from complete.
-  We can't put the entire app into a single component.
-  We need to break it up into sub-components and teach them to work together
-  as we learn in the [next chapter](toh-pt3.html).
+  ## The road ahead
+  Your Tour of Heroes app has grown, but it's far from complete.
+  You can't put the entire app into a single component.
+  <!-- CF: It's not clear to me why you can't put it into a single component. Is this sentence necessary? -->
```

Comment:
As the app grows (especially in the case of a real world production app), the components can get really big and sometimes one can need changes that another doesn't, so if you separate them from one another, you have a lot more flexibility and maintenance is easier. You also have an easier time troubleshooting bugs because you're likely to be able to narrow it down to a specific chunk of code. In this particular example, the app is very small so technically, here, it's not that big of a deal to keep it in one component, but as an app grows, it becomes of paramount importance. Part of what we're doing in the docs is guiding the structure of the project toward that more flexible solution. Does that help?